proto/jumpstarter/v1/jumpstarter.proto

```diff
@@ -127,6 +127,13 @@ message GetReportResponse {
   // jumpstarter.dev/hostname=
   // jumpstarter.dev/name=
   repeated DriverInstanceReport reports = 3;
+  repeated Endpoint alternative_endpoints = 4;
+}
+
+message Endpoint {
+  string endpoint = 1;
+  string token = 2;
+  optional string certificate = 3;
```

Comment:
Nice, since you discover this self-signed cert during connection thorough the main jumpstarter-controller... you know you can trust it. you obtained it in a secure channel :D

And same for the token, 

brilliant!